SRC/interpreter/OpenSeesOutputCommands.cpp

```diff
@@ -386,10 +386,16 @@ int OPS_eleResponse()
     numdata = OPS_GetNumRemainingInputArgs();
     if (numdata > 0) {
 	const char** argv = new const char*[numdata];
+	char buffer[128];
 	for (int i=0; i<numdata; i++) {
-	    argv[i] = OPS_GetString();
+	  argv[i] = new char[128];
```

Comment:
You don't have this new char. The return value from OPS_GetStringFromAll is actually a pointer to buffer passed in.